devenv.lock

```diff
@@ -141,11 +141,11 @@
     },
     "nixpkgs-stable": {
       "locked": {
-        "lastModified": 1711460390,
+        "lastModified": 1710838473,
```

Comment:
feels like this went back in time?

Reply:
You're correct,  it's far too easy to shoot oneself in the foot sometimes on `GitHub` - no changes identified in "Files" tab ...  

I need to reset on top of the latest `main`,  I did so on top of my fork's `main` which was outdated

Reply:
I presume you'll squash this into one commit if it's merged? In which case I can squash it on main!

Reply:
Please squash it so I can just merge :)

Reply:
Done, thanks @domenkozar & nice spot :)